apps/jenkins/jenkins/sbox-intsvc/jenkins.yaml

```diff
@@ -118,17 +118,24 @@ spec:
                       url: "https://yrk32651.live.dynatrace.com/"
           authentication: |
             jenkins:
-              securityRealm:
-                saml:
-                  binding: "urn:oasis:names:tc:SAML:2.0:bindings:HTTP-Redirect"
-                  displayNameAttributeName: "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name"
-                  groupsAttributeName: "http://schemas.microsoft.com/ws/2008/06/identity/claims/groups"
-                  idpMetadataConfiguration:
-                    period: 15
-                    url: https://login.microsoftonline.com/531ff96d-0ae9-462a-8d2d-bec7c0b42082/FederationMetadata/2007-06/FederationMetadata.xml?appid=001b73bc-0fae-46fa-bb7f-2296a655ba33
-                  maximumAuthenticationLifetime: 7862400
-                  usernameAttributeName: "http://schemas.xmlsoap.org/ws/2005/05/identity/claims/name"
-                  usernameCaseConversion: "none"
+              authorizationStrategy:
```

Comment:
this should be updating the existing `authorizationStrategy` not the security realm, you need different changes for the security realm

security realm = authentication
auth strategy = authorisation